faker/providers/__init__.py

```diff
@@ -213,3 +213,17 @@ def bothify(self, text='## ??', letters=string.ascii_letters):
         :returns: string with all numerical and letter placeholders filled in
         """
         return self.lexify(self.numerify(text), letters=letters)
+
+    def hexify(self, text='????', upper=False):
```

Comment:
I would try to come up with a different character than `?` or `#`, maybe `x`?

Reply:
No, `x` is too ASCII, more `$` ?

Reply:
how about `×` (unicode 'multiplication sign')?

Reply:
How do you do it ? 😄

Reply:
good point, i think it's too inconvenient. I'm worried `$` is too close to the idea of 'currency'. Maybe we could use `*`, `^` or `&`

Reply:
What about `~` ?
Easy to do for everyone, without being used all the time.

Reply:
In fact, forget my last comment, let's do `^` !